Dockerfile

```
@@ -0,0 +1,70 @@
+COPY --from=builder /app/remote/package*.json ./remote/
+
+# Copy built files
+COPY --from=builder /app/remote/dist ./remote/dist
+# Copy shared dist into remote/dist/shared to match compiled import paths
+COPY --from=builder /app/shared/dist ./remote/dist/shared
+
+# Install all production dependencies via workspace
+RUN npm ci --omit=dev --ignore-scripts
+
+# Change ownership to non-root user
+RUN chown -R nodejs:nodejs /app
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)